src/keymap.cpp

```diff
@@ -492,6 +492,13 @@ static const std::vector<OpDesc> opdescs = {
 	{OP_INT_SET, "set", "internal-set", "", KM_INTERNAL},
 
 	{OP_INT_GOTO_URL, "gotourl", "internal-goto-url", "", KM_INTERNAL},
+	{
```

Comment:
Please move this block up and put it right after `OP_PREVSEARCHRESULTS`, so the order here matches the order in `enum Operation`.